optaplanner-integration/optaplanner-spring-boot-example/src/main/java/org/optaplanner/spring/boot/example/service/RoomService.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.spring.boot.example.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.optaplanner.spring.boot.example.domain.Room;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RoomService {
```

Comment:
In spring, we can avoid having to crate these services by exposing the RoomRepository directly as a REST service.

Reply:
Less code is better, as the aim is to build this example from scratch during presentations. If we can reuse the spring approach to just expose the RoomRepository, LessonRepository and TimeslotRepository as REST services automatically (which have pagination btw), which having to add additional classes, that would make a big difference.